libdrgn/register_state.c

```diff
@@ -31,6 +31,27 @@ struct drgn_register_state *drgn_register_state_create_impl(uint32_t regs_size,
 	return regs;
 }
 
+struct drgn_register_state *drgn_register_state_dup(
+		const struct drgn_register_state *regs)
+{
+	uint32_t bitset_size = ((uint32_t)regs->num_regs + CHAR_BIT + 1) / CHAR_BIT;
+	size_t size = 0;
```

Comment:
No need to initialize this, it's overwritten in a couple of lines.
```suggestion
	size_t size;
```